pom.xml

```diff
@@ -153,6 +153,7 @@
           </execution>
           <execution>
             <id>yarn install</id>
+            <phase>pre-clean</phase>
```

Comment:
I think this will be a problem, it means `mvn install` will not install yarn deps